src/functions/on-call/list-on-call-data/handler.ts

```diff
@@ -28,18 +25,33 @@ export const listOnCallDataHandler: ValidatedEventAPIGatewayProxyEvent<any> = as
     }
   }
 
-  const s3 = new S3();
+  const dynamoDb = new DynamoDB.DocumentClient();
 
-  const nameMapFile = "name-map.json";
-  const yearFile = `${year}.json`;
-  const paidFile = "paid.json";
+  // Получаем все записи за указанный год
+  const dataResult = await dynamoDb.query({
```

Comment:
Can you explain why we now need 2 data base tables for listing the on call data? If all we need is the persons name, paid boolean and I guess dates- it seems one table should be fine.

As far as I can tell you are also missing a model (see database/models) to describe the on call data.
It would be good if we had a type/ model to describe the dataResult and nameMapResult as currently they are both typed as any